src/ast_visitor.ml

```diff
@@ -61,6 +62,8 @@ class ['s] ast_mapper = object(self)
     let e1', st1 = self#expr e1 st in
     let e2', st2 = self#expr e2 st1 in
     EBankedAA (id, e1', e2'), st2
+  method private eview (id, params) st =
+    EView (id, params), st
```

Comment:
This seems incorrect. You would want to visit the `expr`s in the list with `elist_visit` method and use that to recreate the `EView`.